zebedee-reader/src/test/java/com/github/onsdigital/zebedee/reader/api/endpoint/PublishedDataTest.java

```diff
@@ -0,0 +1,92 @@
+package com.github.onsdigital.zebedee.reader.api.endpoint;
+
+import com.github.onsdigital.zebedee.exceptions.BadRequestException;
+import com.github.onsdigital.zebedee.exceptions.ZebedeeException;
+import com.github.onsdigital.zebedee.reader.FakeCollectionReaderFactory;
+import com.github.onsdigital.zebedee.reader.ZebedeeReader;
+import com.github.onsdigital.zebedee.reader.api.endpoint.PublishedData;
+import com.github.onsdigital.zebedee.reader.configuration.ReaderConfiguration;
+import com.github.onsdigital.zebedee.reader.data.language.ContentLanguage;
+import org.apache.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Date;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+
+public class PublishedDataTest {
```

Comment:
Continuing on from my previous comment about collection IDs - Once you have confirmed the behaviour/requirements can you add another test case to cover the scenario where there is a `collectionID` in the request please?

Depending on what the requirements are this will either verify the expected exception/error status is returned OR it returns the expected content.

Reply:
see Jess's comment for collectionID